src/components/DropZone/utils/index.ts

```diff
@@ -57,3 +57,9 @@ function isChangeEvent(
 ): event is React.ChangeEvent<HTMLInputElement> {
   return Object.prototype.hasOwnProperty.call(event, 'target');
 }
+
+export const defaultAllowMultiple = true;
+
+export function pluralSuffix(isPural: boolean) {
+  return isPural ? 's' : '';
+}
```

Comment:
I think this could get merged if we make the change to remove the letter `s` appending and organize the translations differently. Appending the letter `s` would break in some languages. @sylvhama do you have time to tackle this ? I can try to push some changes to have this fixed :)